src/formatter/codedata/ParsedCode.hx

```diff
@@ -241,6 +241,9 @@ class ParsedCode {
 			if (~/^\s*$/.match(line)) {
 				emptyLines.push(index);
 			}
+			if (line.startsWith("<<<<<<<")) {
```

Comment:
In theory there could probably be false positives with this in multiline string literals?

Reply:
technically yes, there could be false positives...
I could use tokentree to find `<<` that have a single child of `<<`, but since it's not guaranteed that a merge conflict won't occur inside a string literal or a comment, there is no perfect solution

Reply:
>it's not guaranteed that a merge conflict won't occur inside a string literal or a comment,

Right, I didn't think of that edge case... :D